setup.py

```diff
@@ -27,7 +27,7 @@
       use_scm_version=True,
       install_requires=[
           'ruamel.yaml>=0.15.45', 'numpy>=1.17', 'scipy', 'matplotlib', 'utm', 'pymap3d', 'ordered-set',
-          'setuptools>=42', 'rtree',
+          'setuptools>=42', 'rtree', 'opencv-python'
```

Comment:
I think this may be better in `extra_requires`. There are examples for this for `video`/`tensorflow`.
You can raise a warning on the relevant imports, like the tensorflow notebooks.

https://github.com/dstl/Stone-Soup/blob/c806374c4071071919be1b43cbe83db286873df7/stonesoup/detector/tensorflow.py#L5-L13

Reply:
@sdhiscocks I'll be picking up this PR, since Charlotte has now finished her internship. Do you think this should go under the video extras?

Reply:
Either _video_ or many a new group. _video_ may be best, so we don't have too many, and it makes sense for `opencv`